### ISSUE TITLE
Allow notifying that the site will probably not work with the old browser

### ISSUE THREAD

**Issue by wmertens** [browser-update/browser-update#270]
I would like a configuration option to tell users that their browser is not supported on this page and that the site will most likely not work properly.

The current message only suggests updating...

**Comment by josselex** [comment]
What text message would you suggest?

**Comment by wmertens** [comment]
"Your browser is too outdated to use this website. Some functionality will be broken. Please update or use a different browser."

I can provide the Dutch translation :-)

**Comment by alistaircom** [comment]
I can see this being useful. There would be two levels of warning. The first level is for browsers that are out of date but still compatible.  The second warning would be for browsers that are not compatible, and will not work.  Each of these messages can be customized by text and by class, giving a different experience for me. However, this will add some overhead to the script.

**Comment by alistaircom** [comment]
It would also be important that the messages are mutually exclusive, users should only get one of the two messages, never both.

**Comment by josselex** [comment]
This is a good idea. I'm thinking about a good/simple way to implement this.

**Comment by wmertens** [comment]
While a two-level warning would be more accurate, it is very difficult to do, since you would have to enumerate the newly used features and mark them as crucial vs optional, and then they need to be tested.

Does it add much to the user experience? Wouldn't being told that the browser will likely have problems be enough?

**Comment by alistaircom** [comment]
You could write in a fail condition that if the browser were to fail any particular test (i.e.: a browser older than version X or doesn't have some feature) then you'd stop counting and trigger the critical notice.

**Comment by kaptainkommie** [comment]
I agree. I like this idea as well. I would like to warn visitors to my site that much of the functionality and many of the images will not load properly when using IE < 12.

It need not be a browser feature check in the code, that is way too cumbersome and hard to maintain. 

Just allow us to specify through configuration options which browser brands/versions to show the "functionality broken" warning and another configuration array for showing just the update warning?
